openapi/schemas.yaml

```diff
@@ -42,10 +42,11 @@ components:
       type: string
       description: The URL of the incoming payment or ILP STREAM connection that is being paid.
       format: uri
-      pattern: '^https://(.+)/(incoming-payments|connections)/[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}$'
+      pattern: '^(https|http)://(.+)/incoming-payments/(.+)$'
```

Comment:
As for the id change:

1. figured it was correct to loosen it to any sort of id (not necessarily uuid) after discussing with Max. Thinking was that while we use uuid's in rafiki this is really a detail which might vary from implementation to implementation and it shouldn't (I think) matter what sort of id scheme the api implementation uses. More opinions welcome.

2. I changed it to match anything but perhaps chars/numbers only would be better? I just wasn't sure about forbidding the use of query variables. While we don't support anything like this I don't know that we should have an opinion against it: `incoming-payments/123?something=1`

Reply:
I think matching anything is fine in this case